### ISSUE TITLE
Better error message for: "Uncaught Error: Assertion Failed: Value must be a string or undefined, number given"

### ISSUE THREAD

**Issue by gossi** [jelhan/ember-style-modifier#9]
I had this object:

```
elementStyle = {
    position: 'fixed',
    bottom: 0,
    'margin-bottom': 0,
    width: '100%'
};
```

Which I passed to the `{{style}}` modifier `<div {{style this.elementStyle}}>`. That gave me the error message:

```
Uncaught Error: Assertion Failed: Value must be a string or undefined, number given
```

Where my initial thought was: "hey... I passed an object, neither string nor number - what's happening". Actually, it was complaining about `0` in `bottom: 0` not beeing a string but a number. Maybe `0` is special ;) however the error message could be way better, should at least say "the value for rule xyz" 👍

**Comment by jelhan** [comment]
Thanks for reporting. #10 should fix this. Afterwards assertion would look like `Value must be a string or undefined, number ("0") given for property bottom`. Do you see additional improvements?

**Comment by gossi** [comment]
Turn the sentence around, which is better known at first:

`Your given value for property 'bottom' is `0` (number). Accepted types are string and undefined. Please change it.`

That'll give the context first, which gives the developer a "home" situation and kindly ask to change it :)

**Comment by jelhan** [comment]
I like that one. :100: Updated PR accordingly.
